src/amd_layout.js

```diff
@@ -1,9 +1,16 @@
 (function(global, factory) {
-  if (typeof define === 'function' && define.amd)
+  if (typeof module === "object" && typeof module.exports === "object" )
+    module.exports = global.document
+      ? factory(global, true)
+      : function(w) {
```

Comment:
This paragraph is come from [jquery/src/wrapper.js](https://github.com/jquery/jquery/blob/master/src/wrapper.js), I don't know if there is a better way to do that.